x-pack/plugins/ml/public/explorer/explorer_charts/components/explorer_chart_label/explorer_chart_label.js

```diff
@@ -35,15 +35,16 @@ export function ExplorerChartLabel({ detectorLabel, entityFields, infoTooltip, w
   const entityFieldBadges = entityFields.map((entity) => {
     return (
```

Comment:
Nit.. Could just use implicit return here since we're only returning the fragment.